src/templates/udid_config.twig

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>PayloadContent</key>
+        <dict>
+            <key>URL</key>
+            <string>http://{{ appHost }}/udid/submit</string>
+            <key>DeviceAttributes</key>
+            <array>
+                <string>UDID</string>
+                <string>DEVICE_NAME</string>
+                <string>VERSION</string>
+                <string>PRODUCT</string>
+            </array>
+        </dict>
+        <key>PayloadOrganization</key>
+        <string>{{ organization }}</string>
+        <key>PayloadDisplayName</key>
+        <string>{{ organization }} UDID Discovery</string>
+        <key>PayloadVersion</key>
+        <integer>1</integer>
+        <key>PayloadUUID</key>
+        <string>1F996AF4-AA72-43D8-BC7C-44B78AA9630E</string>
```

Comment:
As discussed with iOS developers, this value should be dynamic.

Reply:
@mrix, added. Thx for comment